patches/minecraft/net/minecraft/server/dedicated/DedicatedServer.java.patch

```diff
@@ -42,3 +50,7 @@
              }
          }
      }
++    public void func_145747_a(net.minecraft.util.IChatComponent message) { field_155771_h.info(message.func_150254_d()); }
```

Comment:
Not sure if this is the correct way to make that work, this will log the message using legacy color codes instead of unformatted, however changing it in `MinecraftServer` directly would also affect the integrated server on the client.